### ISSUE TITLE
Missing update events from socket API

### ISSUE THREAD

**Issue by alexandrzhulev** [etherdelta/etherdelta.github.io#371]
Hello @zackcoburn !

I have faced with following issue while using API: 
- initially after connecting to the socket - I receive actual scope of orders;
- as I am subscribed to 'orders' event - further I receive updates of orders ;
- later due to unknown reasons - socket doesn't return 'orders' updates, though I definetly know that they took place. I have rechecked this by placing order to the order book and noticing it on dashboard - but no event from socket regarding 'orders' update is fired;
- though further I could receive 'orders' event - but the updates doesn't include orders that I have already missed;
- besides, there could be the case when socket is temporary disconnected and then connected at once. I assume that at this period I am also missing some updates from the API.
To summarize - after such flow I have not actual order book - so it differs from EtherDelta.

Could this issue be handled (prevented) on EtherDelta socket side ? Are there any solutions how it could be handled on my side (without any extra request to the socket) ?
